Reference Path not recognized in latest 4.0.5 version

In the latest release @types have been included . but the reference path in `built/index.d.ts` is not correct which causes the below error on compiling -

```
error TS6053: File '/Users/pasalar/Documents/coding/testtypescript/node_modules/@types/node/index.d.ts' not found.
error TS6053: File '/Users/pasalar/Documents/coding/testtypescript/node_modules/@types/jasmine/index.d.ts' not found.
```

I fixed it by changing the reference path to- 

```
/// <reference path="../node_modules/@types/node/index.d.ts" />
/// <reference path="../node_modules/@types/jasmine/index.d.ts" />
```

--- a/exampleTypescript/tsconfig.json
+++ b/exampleTypescript/tsconfig.json
@@ -6,7 +6,8 @@
     "sourceMap": false,
     "declaration": false,
     "noImplicitAny": false,
-    "outDir": "tmp"
+    "outDir": "tmp",
+    "types": ["node", "jasmine"]
   },
   "exclude": [
     "node_modules",

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -33,7 +33,7 @@ var runSpawn = function(done, task, opt_arg, opt_io) {
 };
 
 gulp.task('built:copy', function() {
-  return gulp.src(['lib/**/*','!lib/**/*.ts'])
+  return gulp.src(['lib/**/*.js','lib/globals.d.ts'])
       .pipe(gulp.dest('built/'));
 });
 
@@ -89,9 +89,8 @@ gulp.task('types', function(done) {
     'config', 'plugins', 'ptor'];
   var outputFile = path.resolve(folder, 'index.d.ts');
   var contents = '';
-  contents += '/// <reference path="../../@types/node/index.d.ts" />\n';
-  contents += '/// <reference path="../../@types/jasmine/index.d.ts" />\n';
   contents += '/// <reference path="../typings/index.d.ts" />\n';
+  contents += '/// <reference path="./globals.d.ts" />\n';
   contents += 'import {ActionSequence, By, WebDriver, WebElement, WebElementPromise, promise, promise as wdpromise, until} from \'selenium-webdriver\';\n';
   files.forEach(file => {
     contents += parseTypingsFile(folder, file);

--- a/spec/install/tsconfig.json
+++ b/spec/install/tsconfig.json
@@ -5,7 +5,8 @@
     "moduleResolution": "node",
     "sourceMap": false,
     "declaration": false,
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "types": ["node", "jasmine"]
   },
   "exclude": [
     "node_modules",

